### PR TITLE
🌑 DARK MODE & Other UI Improvements

### PR DESCRIPTION
**Pull request by EvanDarwin**
This pull request introduces a dark mode theme for txAdmin. It can be enabled through a toggle that will be in the top of the navbar. This also includes a series of UI improvements for the light theme as well, for a simpler experience.

**Changes & Improvements:**
- Added a dark mode toggle
- Bolded the txAdmin and FXServer version numbers in the footer
- The footer is now static at the bottom of the page
- All pages now use the nicer looking slider now
- Tweaked the "deployment type" cards on the recipes page to work with dark mode
- Fixed a bug with elements not being centered on the resources page

**TODO:**
- [x] Actual toggle client JS
  - [x] Toggle elements working
  - [x] Web Storage for saving their option
- [x] Clean up CSS/changes
- [x] Get the navbar statuses working
- [x] Add cute goat to SCSS file 🐐

### CODE REVIEW

File: web/basic/header.html [hunk 1]
```diff
@@ -190,6 +191,15 @@
                 <img class="c-sidebar-brand-minimized" src="img/tx.png" width="30" height="30" alt="txAdmin Logo">
             </a>
             <ul class="c-header-nav ml-auto mr-4">
+                <a class="c-header-nav-item">
```
Comment (EvanDarwin) [inline web/basic/header.html]: WIP, broken currently

File: web/basic/header.html [hunk 2]
```diff
@@ -190,6 +191,15 @@
...
+                    <svg class="c-sidebar-nav-icon" id="darkToggleLight">
+                        <use xlink:href="img/coreui_icons-full.svg#cil-sun"></use>
```
Comment (TasoOneAsia) [inline web/basic/header.html]: There is no longer a need for the xlink namespace in SVG 2, it is deprecated. You can just use `href` in its place.

See [here](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/xlink:href)
```suggestion
                        <use href="img/coreui_icons-full.svg#cil-sun"></use>
```

File: web/basic/header.html [hunk 3]
```diff
@@ -190,6 +191,15 @@
...
+                    </svg>
+                    <svg class="c-sidebar-nav-icon" id="darkToggleDark" style="display:none">
+                        <use xlink:href="img/coreui_icons-full.svg#cil-moon"></use>
```
Comment (TasoOneAsia) [inline web/basic/header.html]: Same as previous comment.
```suggestion
                        <use href="img/coreui_icons-full.svg#cil-moon"></use>
```

**Review by EvanDarwin — COMMENTED** [review]
I need some help getting the entry in the navbar centered. @TasoOneAsia ?

**Comment by TasoOneAsia** [comment]
> I need some help getting the entry in the navbar centered. @TasoOneAsia ?

Will take a look in the morning.

File: web/basic/header.html (hunk 1 above)
Comment (EvanDarwin) [reply web/basic/header.html]: Resolved

**Review by TasoOneAsia — COMMENTED** [review]
👍 lgtm

**Pull request merged**